zkvm/docs/zkvm-spec.md

```diff
@@ -1298,29 +1314,33 @@ Fails if the `prevoutput` is not a [data type](#data-type) with exact encoding o
 _items... predicate_ **output:_k_** → ø
 
 1. Pops [`predicate`](#predicate) from the stack.
-2. Pops `k` items from the stack.
+2. Pops `k` [portable items](#portable-types) from the stack.
```

Comment:
If the top `k` items includes a non-portable item, is that an error or is it skipped? I know you intend it to be an error, but I think it's a little ambiguous as written.

I think the original wording ("Pops `k` items from the stack") plus the new "fails if" text below is fine.